### DISCUSSION TITLE
请教HME100K数据集合并的问题

### DISCUSSION THREAD

**Discussion by yazheng0307** [SWHL/TrOCR-Formula-Rec#6] [Q&A]
请问下载好HME100K数据集数据集后，直接把train_images的图像复制到原来的images目录下就可以了吗，我这边下载的HME100K图像训练集都是以train_开头。

**Comment by SWHL** [comment]
没看懂，你这问题和我这项目没有关系啊

**Reply by yazheng0307** [reply]
是这样的，[UniMER_Dataset](https://huggingface.co/datasets/wanderkid/UniMER_Dataset)提供的数据里面没有HME100K 的图像，需要单独下载，下面是数据集的说明：
![image](https://github.com/user-attachments/assets/baf7826d-7011-4946-a954-842ae10c82be)
我的问题意思是，单独下载好HME100K数据集后，是不是把HME100K中的训练图像复制到 UniMER_Dataset下的images里面就行了？或者您对于HME100K数据是如何加进来的？

**Reply by SWHL** [reply]
我去，我一直以为他们发布的是全部的呢！！！我没加。。。。。。
这个如果要加的话，需要更改文件名，对应的label需要追加到现在txt最后，保证索引能对上
欢迎一起来搞

**Reply by yazheng0307** [reply]
可以，这是我邮箱：981638732@qq.com
